lib/matplotlib/tests/test_mathtext.py

```diff
@@ -371,3 +371,14 @@ def test_math_fontfamily():
              size=24, math_fontfamily='dejavusans')
     fig.text(0.2, 0.3, r"$This\ text\ should\ have\ another$",
              size=24, math_fontfamily='stix')
+
+
+def test_mathtext_cmr10_minus_sign():
+    # cmr10 does not contain a minus sign and used to issue a warning
+    # RuntimeWarning: Glyph 8722 missing from current font.
+    mpl.rcParams['font.family'] = 'cmr10'
+    fig, ax = plt.subplots()
+    ax.plot(range(-1, 1), range(-1, 1))
+    with pytest.warns(None) as record:
+        fig.canvas.draw()
+    assert len(record) == 0
```

Comment:
For some reason, the tests fail this. To have more insight, you could do:
```suggestion
    assert len(record) == 0, (f"Got {len(record)} warnings:\n" + "\n".join(
                                  str(e.message) for e in record))
```

Reply:
```suggestion
assert len(record) == 0, "\n".join(str(e.message) for e in record)
```